test/test_fork.py

```diff
@@ -235,3 +235,12 @@ def test_qualnames(py, austin):
 
     assert has_pattern(result.stdout, "qualnames.py:Foo.run"), compress(result.stdout)
     assert has_pattern(result.stdout, "qualnames.py:Bar.run"), compress(result.stdout)
+
+
+@allpythons()
+def test_no_logging(py):
+    datafile = tmp_path / "test_no_logging.austin"
+    env = {"AUSTIN_NO_LOGGING", ""}
+    script = "import time; time.sleep(.1)"
+    result = austin("-i", "1ms", "-o", datafile, *python(py), "-c", script, env=env)
```

Comment:
```suggestion
def test_no_logging(py, tmp_path, monkeypatch):
    monkeypatch.setenv("AUSTIN_NO_LOGGING", "1") 
    result = austin("-i", "1ms", *python(py), target("target34.py"))
    assert has_pattern(result.stdout, "target34.py:keep_cpu_busy:3"), compress(
        result.stdout
    )
```
Bonus if we could also test that there are no Austin messages in logs (in a platform-specific way) :slightly_smiling_face: .

It would also be great if we could run this test as additional parameters for existing tests, but this can be addressed with a separate PR.

Reply:
OK, I'm not so familiar with monkeypatch - my initial test function was based on a brief reading of the other tests in test_fork.py, so I'm happy that you could improve it.